src/pg/sql/41_observatory_augmentation.sql

```diff
@@ -0,0 +1,995 @@
+                 'population_3_years_over',
+                 'in_school',
+                 'in_grades_1_to_4',
+                 'in_grades_5_to_8',
+                 'in_grades_9_to_12',
+                 'in_undergrad_college',
+                 'pop_25_years_over',
+                 'high_school_diploma',
+                 'less_one_year_college',
+                 'one_year_more_college',
+                 'associates_degree',
+                 'bachelors_degree',
+                 'masters_degree',
+                 'pop_5_years_over',
+                 'speak_only_english_at_home',
+                 'speak_spanish_at_home',
+                 'pop_determined_poverty_status',
+                 'poverty',
+                 'median_income',
+                 'gini_index',
+                 'income_per_capita',
+                 'housing_units',
+                 'vacant_housing_units',
+                 'vacant_housing_units_for_rent',
+                 'vacant_housing_units_for_sale',
+                 'median_rent',
+                 'percent_income_spent_on_rent',
+                 'owner_occupied_housing_units',
+                 'million_dollar_housing_units',
+                 'mortgaged_housing_units',
+                 'pop_15_and_over',
+                 'pop_never_married',
+                 'pop_now_married',
+                 'pop_separated',
+                 'pop_widowed',
+                 'pop_divorced',
+                 'commuters_16_over',
+                 'commute_less_10_mins',
+                 'commute_10_14_mins',
+                 'commute_15_19_mins',
+                 'commute_20_24_mins',
+                 'commute_25_29_mins',
+                 'commute_30_34_mins',
+                 'commute_35_44_mins',
+                 'commute_45_59_mins',
+                 'commute_60_more_mins',
+                 'aggregate_travel_time_to_work',
+                 'income_less_10000',
+                 'income_10000_14999',
+                 'income_15000_19999',
+                 'income_20000_24999',
+                 'income_25000_29999',
+                 'income_30000_34999',
+                 'income_35000_39999',
+                 'income_40000_44999',
+                 'income_45000_49999',
+                 'income_50000_59999',
+                 'income_60000_74999',
+                 'income_75000_99999',
+                 'income_100000_124999',
+                 'income_125000_149999',
+                 'income_150000_199999'
+  ];
+
+  q = 'WITH a As (
+         SELECT
+           dimension As names,
+           dimension_value As vals
+        FROM OBS_GetCensus($1,$2)
+      )' ||
+      OBS_BuildSnapshotQuery(target_cols) ||
+      ' FROM  a';
+
+  RETURN QUERY
+  EXECUTE
+    q
+  USING geom, target_cols;
+
+  RETURN;
+END;
+$$ LANGUAGE plpgsql;
+
+
+--Creates a table with the young family segment.
+
+CREATE OR REPLACE FUNCTION OBS_GetSegmentFamiliesWithYoungChildren(address text)
+RETURNS TABLE (
```

Comment:
there are two things that worry me here: using address as input and returning a table of unknown size.

cc/ @andrewxhill @javisantana